### PR TITLE
Mark WarningHeader feature gate as GA

### PR DESCRIPTION
**Pull request by liggitt**
xref https://github.com/kubernetes/enhancements/issues/1693

### CODE REVIEW

**Comment by PI-Victor** [comment]
really appreciate it, thank you!

/assign

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -181,6 +181,7 @@ different Kubernetes components.
 | `ValidateProxyRedirects` | `true` | Beta | 1.14 | |
 | `VolumeCapacityPriority` | `false` | Alpha | 1.21 | - |
 | `WarningHeaders` | `true` | Beta | 1.19 | |
+| `WarningHeaders` | `true` | GA | 1.22 | |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:184]: I'm afraid you need to manually move this to the other (graduated features) table.

**Review by sftim — DISMISSED** [review]

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (liggitt) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:184]: oops, done

**Comment by sftim** [comment]
LGTM
Tech review needed, but should be trivial.

**Comment by liggitt** [comment]
/hold cancel

https://github.com/kubernetes/kubernetes/pull/100754 is merged

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-28032--kubernetes-io-vnext-staging.netlify.app/docs/reference/command-line-tools-reference/feature-gates/

/approve

**Pull request merged**